packages/skeleton/src/lib/components/ListBox/ListBox.svelte

```diff
@@ -35,9 +39,11 @@
 	export let labelledby = '';
 
 	// Context
+	setContext('disabledItems', disabled);
```

Comment:
Please ensure that the context key matches the context value to ensure consistency in styling.

Reply:
done